ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -718,6 +718,14 @@
     "FieldType": "C2ApplicationTypeSelection",
     "SecurityClassification": "Public"
   },
+  {
```

Comment:
Do you think we should move all these links into their own json file? Might make navigation of the main one slightly easier.

Reply:
yeah we should, I could make a ticket for that

Reply:
Made a ticket https://tools.hmcts.net/jira/browse/FPLA-2079